packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -1594,6 +1596,32 @@ suite('lit-html', () => {
       assert.isOk(event);
     });
 
+    test('directives have access to renderOptions', () => {
```

Comment:
This is just testing attribute directives, so add "attribute" to the name.

Should we pass options to NodeParts too?

Reply:
It also tests NodeParts

Reply:
oops!